src/authz-module/components/AuthZTitle.tsx

```diff
@@ -0,0 +1,50 @@
+import { ReactNode } from 'react';
+import {
+  Breadcrumb, Col, Container, Row, Button, Badge,
+} from '@openedx/paragon';
+
+interface BreadcrumbLink {
+  label: string;
+  to?: string;
+}
+
+interface Action {
+  label: string;
+  onClick: () => void;
+}
+
+export interface AuthZTitlePtops {
+  activeLabel: string;
+  pageTitle: string;
+  pageSubtitle: string | ReactNode;
+  navLinks?: BreadcrumbLink[];
+  actions?: Action[];
+}
+
+const AuthZTitle = ({
+  activeLabel, navLinks = [], pageTitle, pageSubtitle, actions = [],
+}: AuthZTitlePtops) => (
+  <Container className="p-5 bg-light-100">
+    <Breadcrumb
+      links={navLinks}
+      activeLabel={activeLabel}
+    />
+    <Row className="mt-4">
+      <Col xs={12} md={8} className="mb-4">
+        <h1 className="text-primary">{pageTitle}</h1>
+        {typeof pageSubtitle === 'string'
+          ? <h3><Badge className="py-2 px-3 font-weight-normal" variant="light">{pageSubtitle}</Badge></h3>
+          : pageSubtitle}
+      </Col>
+      <Col xs={12} md={4}>
+        <div className="d-flex justify-content-md-end">
+          {
+            actions.map(({ label, onClick }) => <Button onClick={onClick}>{label}</Button>)
```

Comment:
I think we could add a key prop here on each element when mapping, this could help React to track of them properly and maybe we should add a type="button" on the Button just to avoid accidental form submissions if this ever ends up inside a form, both are not critical things, just friendly comments that could save headaches later.